bans-core/src/main/java/space/arim/libertybans/core/config/Formatter.java

```diff
@@ -215,12 +228,16 @@ String getVariable() {
 		simpleReplacements.put(SimpleReplaceable.DURATION, durationFormatted);
 		simpleReplacements.put(SimpleReplaceable.START_DATE, formatAbsoluteDate(punishment.getStartDate()));
 		simpleReplacements.put(SimpleReplaceable.TIME_PASSED, formatRelative(timePassed));
+		simpleReplacements.put(SimpleReplaceable.TIME_PASSED_SIMPLE, formatRelativeSimple(timePassed));
 		simpleReplacements.put(SimpleReplaceable.END_DATE, formatAbsoluteDate(punishment.getEndDate()));
 		simpleReplacements.put(SimpleReplaceable.TIME_REMAINING, relativeEndFormatted);
+		simpleReplacements.put(SimpleReplaceable.TIME_REMAINING_SIMPLE, relativeEndFormattedSimple);
+		MessagesConfig.Formatting.PunishmentExpiredDisplay display = messages().formatting().punishmentActiveDisplay();
+		simpleReplacements.put(SimpleReplaceable.HAS_EXPIRED, (display != null) ? ((notExpired) ? display.notExpired() : display.expired()) : "Unknown"); // null check needed for tests
```

Comment:
A better way to make the tests work is to mock the PunishmentExpiredDisplay in the test setup, rather than add a null check to the main code.

If you modify the `setSimpleMessagesFormatting` method in `FormatterTest`, you can do this without much trouble:
```java
	private void setSimpleMessagesFormatting() {
  // These lines already exist
		MessagesConfig.Formatting formatting = mock(MessagesConfig.Formatting.class);
		when(messagesConfig.formatting()).thenReturn(formatting);
		lenient().when(formatting.consoleDisplay()).thenReturn("Console");
		lenient().when(formatting.globalScopeDisplay()).thenReturn("global");
		lenient().when(formatting.punishmentTypeDisplay()).thenReturn(Map.of());
		lenient().when(formatting.noTimeRemainingDisplay()).thenReturn("(No time remaining)");
  // Adding these lines will allow you to remove the null check
		MessagesConfig.Formatting.PunishmentExpiredDisplay expiredDisplay = mock(MessagesConfig.Formatting.PunishmentExpiredDisplay.class);
                lenient().when(expiredDisplay.notExpired()).thenReturn("not expired");
                lenient().when(expiredDisplay.expired()).thenReturn("expired");
                lenient().when(formatting.punishmentExpiredDisplay()).thenReturn(expiredDisplay);
	}
```